addons/website_sale/models/product.py

```diff
@@ -445,13 +474,26 @@ class Product(models.Model):
 
     website_url = fields.Char('Website URL', compute='_compute_product_website_url', help='The full URL to access the document through the website.')
 
+    uom_count = fields.Float('Base Unit Price', default=1)
```

Comment:
base unit price ? not really clear that you expect the count of product :/

Reply:
I agree, that's not what I would name it either but this is part of the specifications:
![specification](https://lh3.googleusercontent.com/YUutbW6uiOkgCatGcy2ODPvQQTuRB4j6jeJZ2bRWLMLT9C_V6-kwjSjMRFHhWXKmu7LC1LraZmzkzrUnVVd9=w1920-h936-rw "Specification")

Should I rename it then regardless of that ?

Reply:
imo bas eunit price is the compute field... not the unit_count field 

@sbuhl @pierre-pde

Reply:
@JKE-be  & @kzhekov , agreed that title is not the best. Anyway, feature doesn't seem to work IMO
[https://bit.ly/3l1bAvZ](https://bit.ly/3l1bAvZ)

Reply:
Yes, I had to change the whole structure to move the new functionality entirely into website_sale, but I broke half of it so now it's fixing time :D

Reply:
@pierre-pde In your example, how is computed the price per unit, $50/kg?

Reply:
This should be fixed as well, as per @pierre-pde's suggestion